alacritty/src/cli.rs

```diff
@@ -322,13 +325,82 @@ pub struct IpcConfig {
     pub reset: bool,
 }
 
-/// Toml value with default implementation.
-#[derive(Debug)]
-pub struct TomlValue(pub Value);
+/// Parsed CLI config overrides.
+#[derive(Debug, Default)]
+pub struct ParsedOptions {
+    config_options: Vec<(String, Value)>,
+}
+
+impl ParsedOptions {
+    /// Parse CLI config overrides.
+    pub fn from_options(options: &[String]) -> Self {
+        let mut config_options = Vec::new();
+
+        let logger_started = logging::is_initialized();
+
+        for option in options {
+            let parsed = match toml::from_str(option) {
+                Ok(parsed) => parsed,
+                Err(err) => {
+                    if logger_started {
+                        error!("Ignoring invalid CLI option '{option}': {err}");
```

Comment:
I'm not really a fun of logging like that, because it'll be visible for all windows
and there's no clear way to solve this issue. And ti's not like we're preventing the window from being spawned in the first place like that.

Thus, I think it'll be nice to have this warning only for a newly created window.

Reply:
This is also missing a `target` so there's no way to clear messages by invoking `alacritty msg config` again, so the only way to manually `Ctrl + L`.

Reply:
> I'm not really a fun of logging like that, because it'll be visible for all windows

But it affects all windows?

> Thus, I think it'll be nice to have this warning only for a newly created window.

So you want it to **only** write to CLI? I'd be okay with that tbh.

Reply:
>But it affects all windows?

It depends on what happens, since you get this log from `alacritty msg create-window -o font.size=12e`, so it affects only the newly created window, thus only this new window should have this log message.

>So you want it to only write to CLI? I'd be okay with that tbh.

At least in the current state of things.